editor.planx.uk/src/@planx/components/List/schemas/ResidentialUnits.ts

```diff
@@ -0,0 +1,73 @@
+import { Schema } from "../model";
+
+export const ResidentialUnits: Schema = {
+  type: "Residential Units",
+  fields: [
+    {
+      type: "question",
+      data: {
+        title: "What type of change are you making?",
+        fn: "changeType",
+        options: [
+          { id: "proposed", data: { text: "Proposed" } },
+          { id: "existing", data: { text: "Existing" } },
+        ],
+      },
+    },
+    {
+      type: "question",
+      unique: true,
+      data: {
+        title: "What is the tenure type?",
+        fn: "tenureType",
+        options: [
+          { id: "marketHousing", data: { text: "Market housing" } },
+          {
+            id: "socialAffordableRent",
+            data: { text: "Social and affordable rent" },
+          },
+          {
+            id: "affordableHomeOwnership",
+            data: { text: "Affordable home ownership" },
+          },
+          { id: "starterHome", data: { text: "Starter homes" } },
+          { id: "selfBuild", data: { text: "Self build" } },
+        ],
+      },
+    },
+    {
+      type: "question",
+      data: {
+        title: "What is the unit type?",
+        fn: "unitType",
+        options: [
+          { id: "houses", data: { text: "Houses" } },
+          { id: "flats", data: { text: "Flats" } },
+          { id: "bedsits", data: { text: "Bedsits" } },
+          { id: "starterHome", data: { text: "Starter homes" } },
+          { id: "shelteredHousing", data: { text: "Sheltered housing" } },
+          { id: "clusteredFlats", data: { text: "Clustered flats" } },
+          { id: "other", data: { text: "Other" } },
+        ],
+      },
+    },
+    {
+      type: "number",
+      data: {
+        title: "How many bedrooms are there?",
+        fn: "numberBedrooms",
+        allowNegatives: false,
+      },
+    },
+    {
+      type: "number",
+      required: false,
+      data: {
+        title: "How many identical units of this type are there?",
```

Comment:
nit: we know from `AddressInput` & `ContactInput` that optional inputs should concat ` (optional)` to their titles for accessibility - can obviously hardcode that now, but may be worthwhile to automatically append based on `required: false` in future?

Reply:
Very good spot 👍 Added in [442427f](https://github.com/theopensystemslab/planx-new/pull/3222/commits/442427f179d1a4fea1eb16b51b87422950b11316)

<img width="453" alt="image" src="https://github.com/theopensystemslab/planx-new/assets/20502206/611d4bb4-5d40-4988-9ea9-48b51ff042bd">